### PR TITLE
[TEST] Add readiness Docker tests

### PR DESCRIPTION
**Pull request by grcevski**
Add new readiness service test inside the DockerTests.

### CODE REVIEW

File: rest-api-spec/build.gradle [hunk 1]
```diff
@@ -81,6 +81,8 @@ tasks.named("yamlRestTestV7CompatTransform").configure { task ->
   task.skipTest("search.aggregation/20_terms/string profiler via global ordinals native implementation", "The profiler results aren't backwards compatible.")
   task.skipTest("search.aggregation/20_terms/string profiler via map", "The profiler results aren't backwards compatible.")
   task.skipTest("search.aggregation/20_terms/numeric profiler", "The profiler results aren't backwards compatible.")
+  task.skipTest("search.aggregation/360_date_histogram/Multi-value date histogram", "The aggregation results aren't backwards compatible.")
```
Comment (rjernst) [inline rest-api-spec/build.gradle:84]: This should be in a separate PR, if we are going to mute it (though the message doesn't seem write, I don't see why these aggs would not be backward compatible, that seems like a bug).

**Review by rjernst — APPROVED** [review]
LGTM

File: rest-api-spec/build.gradle (hunk 1 above)
Comment (rjernst) [reply rest-api-spec/build.gradle:84]: I muted this https://github.com/elastic/elasticsearch/commit/0ecaceb013ad3632a6e3f450b03c49e0b3f5bfeb

Comment (grcevski) [reply rest-api-spec/build.gradle]: Thanks.

**Comment by grcevski** [comment]
@elasticmachine run elasticsearch-ci/bwc
